app/components/elements/FontStyles.js

```diff
@@ -0,0 +1,205 @@
+  color: ${baseTheme.colors.primaryFont};
+`;
+
+export const Subheading = Styled(H4)`
+  font-size: ${baseTheme.fontSizes[2]}px;
+  line-height: 28px;
+  font-weight: 400;
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  color: ${props => (props.color ? props.color : baseTheme.colors.primaryFont)};
+`;
+
+export const Body2 = Styled(Text)`
+  font-size: ${baseTheme.fontSizes[2]}px;
+  line-height: 28px;
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  color: ${baseTheme.colors.primaryFont};
+  margin-block-start: 0em;
+  margin-block-end: 1em;
+  &.m0 {
+    margin: 0;
+  }
+`;
+
+export const Body1 = Styled(Text)`
+  font-size: ${baseTheme.fontSizes[1]}px;
+  line-height: 24px;
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-bottom: 1em;
+  color: ${props => (props.color ? props.color : baseTheme.colors.primaryFont)};
+`;
+
+export const Caption = Styled(Text)`
+  font-size: ${baseTheme.fontSizes[0]}px;
+  line-height: 20px;
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  color: ${props => (props.color ? props.color : baseTheme.colors.primaryFont)};
+`;
+
+export const NavigationLink = Styled(Link)`
+  font-size: ${baseTheme.fontSizes[1]}px;
+  margin: 0;
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  line-height: 14px;
+  font-weight: 400;
+  color: ${baseTheme.colors.primaryFont};
+  &:hover {
+    color: ${baseTheme.colors.linkFont};
+    box-shadow: none;
+    text-decoration: none;
+  }
+`;
+
+export const DropdownGroupHeader = Styled(Text)`
+  font-size: ${baseTheme.fontSizes[1]}px;
+  margin: 0;
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  line-height: 14px;
+  font-weight: 400;
+  color: ${baseTheme.colors.primaryFont};
+  &:hover {
+    color: ${baseTheme.colors.linkFont};
+    box-shadow: none;
+    text-decoration: none;
+  }
+`;
+
+export const TextLinkDisabled = Styled(Text)`
+  color: ${baseTheme.colors.linkFont};
+  cursor: pointer;
+  line-height: 20px;
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  &:hover {
+    color: ${baseTheme.colors.linkFont};
+    text-decoration: none;
+  }
+`;
+
+export const TextLinkStyle = Styled(Link)`
+  color: ${baseTheme.colors.primaryFont};
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  &:hover {
+    color: ${baseTheme.colors.linkFont};
+    text-decoration: none;
+  }
+`;
+
+export const ButtonFont = Styled(Text)`
+  font-size: ${baseTheme.fontSizes[2]}px;
+  font-weight: 400;
+  line-height: 14px;
+  font-family: BasisMedium, Helvetica, Ariel, sans-serif;
+  padding: ${baseTheme.space[0]}px ${baseTheme.space[2]}px;
+  height: auto;
+`;
+
+export const BlockQuote = Styled.blockquote`
+  border-left: ${baseTheme.border};
+  font-weight: normal;
+  margin-left: 0;
+  padding-left: 32px;
+  border-width: 4px;
+  border-color: ${baseTheme.colors.indigo};
+  & div:last-child {
+    margin-bottom: 0;
+  }
+`;
```

Comment:
Tiny nitpick - needs empty line here.

Reply:
noted -- will make sure that's taken care of -- merging now so it's ready for you and Chris in the ET morning